ntbs-service/Migrations/20200227174010_AddPlannedTreatmentRegimen.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class AddPlannedTreatmentRegimen : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "IsMDRTreatment",
+                table: "ClinicalDetails");
+
+            migrationBuilder.DropColumn(
+                name: "IsShortCourseTreatment",
+                table: "ClinicalDetails");
+
+            migrationBuilder.AddColumn<string>(
+                name: "TreatmentRegimen",
+                table: "ClinicalDetails",
+                maxLength: 30,
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "TreatmentRegimenOtherDescription",
+                table: "ClinicalDetails",
+                maxLength: 100,
+                nullable: true);
```

Comment:
We should make sure the existing data doesn't get wiped completely - might get uat users confused.
We should be able to do something along the lines of:
1. create new columns
2. set regiment to standard where IsShortCourseTreatment is true
3. set regiment to mdr where IsMDRTreatment is true
4. drop old columns